examples/ota-provider-app/ota-provider-common/OTAProviderExample.cpp

```diff
@@ -91,26 +93,88 @@ void OTAProviderExample::SetOTAFilePath(const char * path)
     }
 }
 
+void OTAProviderExample::SetOTACandidates(std::vector<OTAProviderExample::DeviceSoftwareVersionModel> candidates)
+{
+    mCandidates = std::move(candidates);
+}
+
+void OTAProviderExample::SetOTACandidates(OTAProviderExample::DeviceSoftwareVersionModel entry)
```

Comment:
Maybe I missed it but I don't think I see this getting called?

Reply:
Left over from a previous commit. I can remove this.